aws_lambda_powertools/metrics/provider/cloudwatch_emf/cloudwatch.py

```diff
@@ -86,6 +88,26 @@ def __init__(
 
         self.dimension_set.update(**self.default_dimensions)
 
+    @staticmethod
+    def is_metrics_disabled() -> bool:
+        """Checks if metrics have been disabled via POWERTOOLS_METRICS_DISABLED or POWERTOOLS_DEV"""
+        # First check POWERTOOLS_METRICS_DISABLED
+        if constants.METRICS_DISABLED_ENV in os.environ:
+            env_value = os.getenv(constants.METRICS_DISABLED_ENV)
+            is_disabled = resolve_truthy_env_var_choice(env=env_value or "false")
+            if is_disabled:
+                logger.debug("Metrics have been disabled via env var POWERTOOLS_METRICS_DISABLED")
+            return bool(is_disabled)
+
+        # Then check if POWERTOOLS_DEV is enabled
+        dev_mode_value = os.getenv(constants.POWERTOOLS_DEV_ENV)
+        dev_mode = resolve_truthy_env_var_choice(env=dev_mode_value or "false")
+        if dev_mode:
+            logger.debug("Metrics have been disabled via env var POWERTOOLS_DEV")
+            return True
+
+        return False
+
```

Comment:
Hi @anafalcao, how about moving this function to the `functions.py` file in the `metrics` folder?